cli/src/main/java/com/devonfw/tools/ide/variable/IdeVariables.java

```diff
@@ -73,6 +75,11 @@ public interface IdeVariables {
   /** {@link VariableDefinition} for {@link com.devonfw.tools.ide.context.IdeContext#getProjectName() PROJECT_NAME}. */
   VariableDefinitionString PROJECT_NAME = new VariableDefinitionString("PROJECT_NAME", null, c -> c.getProjectName());
 
+  /** Preferred Git protocol (HTTPS/SSH) as defined by {@link GitUrlSyntax}. */
+  VariableDefinitionEnum<GitUrlSyntax> PREFERRED_GIT_PROTOCOL = new VariableDefinitionEnum<>("PREFERRED_GIT_PROTOCOL", null, GitUrlSyntax.class,
+      c -> GitUrlSyntax.HTTPS
```

Comment:
The default should be not to change the URL. Only if the user sets the property explicitly, the URL should be changed.
See #625:
> Default behavior remains unchanged if the property is not set, using the provided Git URL as is.

Reply:
Resolved here: https://github.com/devonfw/IDEasy/pull/724#discussion_r1821054880

Reply:
Done in https://github.com/devonfw/IDEasy/pull/724